readme.md

```diff
@@ -1,34 +1,39 @@
-## Gaia Explorer - GEX
+## GEX In-Terminal Gaia Explorer
 
-An in-terminal explorer for Cosmos SDK blockchains.
+GEX is a real time in-terminal explorer for Cosmos SDK blockchains. See the [Check out your Cosmos SDK blockchain in a terminal-based block explorer](https://blog.cosmos.network/gaia-explorer-in-terminal-f37a4ea52e3c) blog post to learn more about GEX.
 
-Displays blocks, transactions, validator, network status and more about peers, connection, version and useful information when you want to have a quick peek into your own node.
-Works with Starport.
+The GEX explorer displays blocks, transactions, validator, network status, and more information. Use the GEX block explorer to see the status of peers, connection, version, and other useful information to have a quick peek into your own node. GEX works with Starport.
 
-## Install
+## Install GEX
+
+The GEX installation requires Go. If you don't already have Go installed, see https://golang.org/dl. Download the binary release that is suitable for your system and follow the installation instructions.
+
+To install the GEX binary:
 
 `go get -u github.com/cosmos/gex`
 
-## Run
+## Run GEX
+
+To launch a GEX explorer in your terminal window, type:
 
 `gex`
 
-## Optional port
+## Optional Port
 
 `gex -p 27657`
 
-## Sneak-Peak
+## Sneak Peak
 
 ![Terminal](./render.gif "Render Terminal")
 
-## Run manually
+## Run Manually
```

Comment:
why would we run GEX manually? if we have a good reason, let's say what that is. otherwise, remove this part

Reply:
I think mostly for developers.

Reply:
Could be part of the CONTRIBUTING file

Reply:
so... I'm not clear on how I can help here... teach me!